starter-code/js/index.js

```diff
@@ -1,11 +1,62 @@
-var $cart = document.querySelector('#cart tbody');
-var $calc = document.getElementById('calc');
+var $calc = document.getElementById("calc");
+var $newprod = document.getElementById("create");
+var $tableBody = document.getElementById("table-body");
 
 function updateSubtot($product) {
   // Iteration 1.1
+  var price = Number($product.querySelector(".pu span").innerHTML);
+  var quantity = Number($product.querySelector(".qty label input").value);
+  var subtotal = Math.round(price * quantity * 100) / 100;
+  $product.querySelector(".subtot").innerHTML = "$" + subtotal;
+  return subtotal;
 }
 
 function calcAll() {
   // Iteration 1.2
+  var $cart = document.querySelectorAll("#cart tbody tr");
+  console.log($cart);
+  var total = 0;
+  $cart.forEach(function(node) {
+    let subTot = updateSubtot(node);
+
+    total += subTot;
+  });
+  document.querySelector("h2 span").innerHTML = total;
+}
+$calc.onclick = calcAll;
+$newprod.onclick = createNewItem;
+$tableBody.onclick = deleteProduct;
+
+function deleteProduct(evt) {
+  if (evt.target.classList.contains("btn-delete"))
+    evt.target.parentElement.parentElement.remove();
+}
```

Comment:
`function deleteProduct(evt) {
  if (evt.target.classList.contains("btn-delete"))
    evt.target.parentElement.parentElement.remove();
    calcAll()
}`

You could call your calcAll function here so the price updates when you delete an element :)